src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -442,7 +447,16 @@ export const generateTelegrafToken = (configID: string) => async (
     dispatch(setToken(createdToken.token))
   } catch (error) {
     console.error(error)
-    dispatch(notify(TokenCreationError))
+
+    if (error.message) {
+      const customNotification = {
```

Comment:
only nitpick is that `customNotification` might be named a little bit more richly and descriptively.

i'm not sure if this structure belongs in `shared/copy/notifications.ts` or should stay here 🤔. It's probably fine here for now since it's not being used by anything else